index.js

```diff
@@ -1,7 +1,15 @@
 'use strict';
 const Build = require('./lib/build');
+const factory = require('./lib/factory');
 const Job = require('./lib/job');
 const Pipeline = require('./lib/pipeline');
 const User = require('./lib/user');
 
-module.exports = { Build, Job, Pipeline, User };
+module.exports = {
+    Build,
+    configureBuildModel: factory.configureBuildModel,
```

Comment:
The more I think about this though, what's the point of using new?  These aren't treated like class instances.

An instance would be like:
```js
const build = new Build(buildId);
build.status = 'FAILED';
build.save(next);
```

If we're going to be treating them statically (like we do now), then let's move it to entirely to a singleton pattern.

```js
const models = require('screwdriver-models');
models.configure({ datastore, executor, key });

models.build.create(payload, next);
```